### PR TITLE
A generated source feature should include the source bundles

### PR DESCRIPTION
**Pull request by merks**
https://github.com/eclipse-tycho/tycho/issues/1648

### CODE REVIEW

**Comment by merks** [comment]
@laeubi My first bug fix!

File: tycho-its/src/test/java/org/eclipse/tycho/test/target/TargetPlatformLocationsTest.java [hunk 1]
```diff
@@ -126,6 +133,31 @@ public void testMavenLocationAutogeneratedFeature() throws Exception {
...
 		verifier.executeGoal("verify");
 		verifier.verifyErrorFreeLog();
+
+		Path targetPlatformRepository = Path.of(verifier.getBasedir(),
+				"test.product/target/targetPlatformRepository/content.xml");
+		try (var stream = Files.newInputStream(targetPlatformRepository)) {
+			Xpp3Dom dom = Xpp3DomBuilder.build(stream, StandardCharsets.UTF_8.displayName());
+			Xpp3Dom[] children = dom.getChild("units").getChildren("unit");
+			Optional<Xpp3Dom> sourceFeature = Stream.of(children)
+					.filter(it -> "org.apache.commons.io.feature.feature.group".equals(it.getAttribute("id")))
```
Comment (laeubi) [inline tycho-its/src/test/java/org/eclipse/tycho/test/target/TargetPlatformLocationsTest.java:143]:
```suggestion
					.filter(it -> "org.apache.commons.io.source..feature.feature.group".equals(it.getAttribute("id")))
```
Probably this one is actually wanted (but just guessing)?

Comment (merks) [reply tycho-its/src/test/java/org/eclipse/tycho/test/target/TargetPlatformLocationsTest.java]: Yep, that's the failing test. 😖

**Comment by merks** [comment]
@laeubi  I think it's good to go now...

**Review by laeubi — APPROVED** [review]
Thanks for the fix!

**Comment by merks** [comment]
I feel so empowered now!  🚀

**Pull request merged**